### DISCUSSION TITLE
Recent update seems to have confused gas tariff config

### DISCUSSION THREAD

**Discussion by SignUp0** [BottlecapDave/HomeAssistant-OctopusEnergy#212] [General]
Thanks again for such a useful integration. I've been using successfully for a few months.
I took a few updates recently, to HA 2023.4.2 and then several jumps in the Octopus integration via HACS.

Somewhere in these upgrades I've picked up a new error
> "Unknown tariff - Gas - G-1R-VAR-22-11-01-H"
> The tariff "G-1R-VAR-22-11-01-H" associated with your Gas meter has not been found. Click on 'Learn More' with instructions on what to do next.

I haven't changed the tariff, looking on the Octopus website account page it's: Flexible Octopus November 2022 v1

From the error it looks like the API is returning valid data relating to the tariff, is there anything I need to do to resolve the error?

Thanks again!

**Comment by BottlecapDave** [comment]
Hi there.

The check hsa been added to try and reduce the raised issues with the OE API that are beyond my control. However if you don't have internet connection (or the API is playing up) when the check is made you can get false positives. I've been seeing sporadic issues today with the API today which might have cause it. I'll see if I can add some additional checks. If you're continuing to see updates, you can choose to ignore the repair error.
